.circleci/config.yml

```diff
@@ -129,6 +129,27 @@ jobs:
               "${ARNOLD_IMAGE}" \
               ansible --version
 
+  test-bootstrap:
+    machine: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - *attach_workspace
+      - *docker_load
+      - *ci_env
+      - run:
+          name: Test deploy in OpenShift
+          command: |
+            sudo bash -c "echo '{\"insecure-registries\": [\"172.30.0.0/16\"]}' > /etc/docker/daemon.json"
+            sudo service docker restart
+            wget https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
+            tar -xf openshift*.tar.gz --strip-components=1
+            export PATH="${PATH}:${PWD}"
+            LOCALE_IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+            oc cluster up --server-loglevel=5 --public-hostname="${LOCALE_IP}"
+            oc login -u developer -p developer https://${LOCALE_IP}:8443 --insecure-skip-tls-verify=true
+            echo "arnold" | USE_TTY=false ./bin/bootstrap -e "apps_filter=richie"
```

Comment:
I think you can split this step in 3 separated steps:

1. Install & configure oc
2. Start the OpenShift cluster
3. Bootstrap the test application